docs/dev/mkdocs.md

```diff
@@ -16,6 +16,9 @@ See [this link](https://mkdocs.github.io/mkdocs-bootswatch/) for a list of boots
 Next, install font awesome for icons to display in the documentation.   
 `pip install fontawesome-markdown`
 
```

Comment:
Can we please make it more clear to the reader that these dependencies are already installed by the conda_installs script. Maybe describing the deps is still worth including, but it's confusing as it's worded right now.

Reply:
That is the first line in this section
**Important: The below installations are added to conda_insalls scripts.**

Reply:
Just fixed the typo in "installs"

Reply:
I just slightly modified this to clarify the packages have already been installed.